core/src/main/java/org/apache/druid/java/util/common/ByteBufferUtils.java

```diff
@@ -128,14 +127,10 @@ private static MethodHandle unmapJava7Or8(MethodHandles.Lookup lookup) throws Re
 
```

Comment:
Java 7 is not supported by Druid, so it doesn't make sense to mention it in the method name.

Reply:
sure, but it's nice to know it works there as well, if someone encounters this code. The name was was ported over from Apache Kafka.

Reply:
Then there should be a Javadoc comment on the method mentioning that. Otherwise, readers of this class may this question and will go re-check if Druid still supports Java 7.

Reply:
done